src/js/video.js

```diff
@@ -749,5 +749,9 @@ videojs.dom = Dom;
  */
 videojs.url = Url;
 
+Object.defineProperty(videojs, 'TERMINATOR', {
```

Comment:
this will break in IE8 :)

Also, I think it's reasonable to make it enumerable and it might be nice to make writeable false be explicit.

Reply:
Is there an example somewhere else in the project of how to do this for IE8 as well? I tried to find one earlier and came up empty.

🆗  explicit `writeable: false` and `enumerable: true` sound good to me

Reply:
something like
```js
if (Object.defineProperty && !browser.IS_IE8) {
  Object.defineProperty(...);
} else {
  videojs.TERMINATOR = TERMINATOR;
}
```